src/main/java/Core/Main.java

```diff
@@ -6,10 +10,13 @@
     public static void main(String[] args) throws IOException, InvocationTargetException, IllegalAccessException {
 
         int portNumber = 5000;
+        String path = "/Users/miss.merce/Workspace/Java/HttpServer/Public/";
```

Comment:
Most users won't have this directory at this filepath on the computer running this software. It feels like a good place to leverage the `args` being passed to the main method.

Reply:
Good one! I guess I could do that with the port as well